pyccel/codegen/printing/ccode.py

```diff
@@ -1592,8 +1592,8 @@ def _print_IndexedElement(self, expr):
             list_var = self._print(ObjectAddress(base))
             container_type = self.get_c_type(base.class_type)
             if assign:
-                assert len(assign) == 1
-                assign_node = assign[0]
+                #assert len(assign) == 1
```

Comment:
Why comment the assert?

Reply:
Hmm, I don't remember. This probably means I had multiple assign nodes. That's usually indicative of a problem with the AST tree. I meant to go back and hunt it down once everything else was working. I have reverted the change: e514f3c3fab9a5722db9f64bccd3fc48a78f0e34
If the tests still pass then it's ok. Otherwise I will have to hunt this down

Reply:
It seems to be fine now